regression/cases/syslog_regex_logs2metric_ddmetrics/vector/vector.toml

```diff
@@ -27,7 +27,7 @@ type = "log_to_metric"
 inputs = ["remap"]
 
   [[transforms.log2metric.metrics]]
-  type = "gauge"
+  type = "counter"
```

Comment:
Changed this from a gauge to a counter to induce the worst case behavior in terms of performance.